### ISSUE TITLE
useColors = true

### ISSUE THREAD

**Issue by gtteamamxx** [WPMGPRoSToTeMa/ChatPrint#9]
https://github.com/WPMGPRoSToTeMa/ChatPrint/blob/93dd63cddbf8f170eef84bc423dc4a5a0d2fbfd6/ChatPrint.inc#L164

You can set useColors = false on 'default' in switch and initializing useColors with default true value

**Comment by WPMGPRoSToTeMa** [comment]
Nope. `useColors` is `true` when string has any color specifier, but it is `false` only when the whole string doesn't contain color specifiers. `default` branch in `switch` corresponds for current character/sequence that isn't color specifier.
